dotenv-linter/tests/common/test_dir.rs

```diff
@@ -14,20 +15,32 @@ use std::str::from_utf8;
 /// Use to test commands in temporary directories
 pub struct TestDir {
     current_dir: TempDir,
+    envs: Option<HashMap<String, String>>,
```

Comment:
```suggestion
    envs: HashMap<String, String>,
```
You can use an empty hash map instead of `Option<HashMap>`.